connect.php

```diff
@@ -3,7 +3,11 @@
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
```

Comment:
1. Delete the overheads! "<<<< HEAD ..."
2. Watch the videos to setup db 
3. Share screenshots of the working website on localhost